### ISSUE TITLE
Avoid parsing XML on every logout request

### ISSUE THREAD

**Issue by paultag** [department-of-veterans-affairs/caseflow#111]
```ruby
      doc = Nokogiri.XML(File.open(ENV['SSOI_SAML_XML_LOCATION'], 'rb'))
```

should likely be punted elsewhere. Perhaps over in `omniauth-saml-va`

**Comment by plusjeff** [comment]
Not sure I understand this ticket. @paultag : Could you add more detail?

**Comment by alex** [comment]
https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/controllers/web_controller.rb#L220-L236

**Comment by paultag** [comment]
@plusjeff yeah, the view parses the xml on logout every time we hit the view -- https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/controllers/web_controller.rb#L224-L240
